docs/API/InstrumentFormat.md

```diff
@@ -287,32 +342,188 @@ A NumericElement represents a numeric data input and has the general form of:
     "Options": {
         /* None currently */
     }
 }
 ```
 
-`Type`: MUST be "score".
+`Type`: MUST be "calc".
 
-`Name`: Required. Follows `PageElement.Name` rules. The Name MAY be used by an
+`Name`: Required. Follows `PageElement.Name` rules. The Name will be used by an
         implementation as a field name to save calculated data into.
 
-`Description`: Optional. Follows `PageElement.Name` rules. If not specified, the
-               score will be displayed with no accompagning text.
+`Description`: Optional. If not specified, the
+               score will be displayed with no accompanying text.
+               
+`Formula`: String. Evaluated using the LORIS Logic Parser syntax.
+           The result is stored as the element's value.
+           
+### 2.1.6 Radio
+
+A radio element represents a set of radio buttons all attributed to the same question.
+A general example is provided below:
+
+```js
+{
+    "Type": "radio",
+    "Name": REQUIRED,
+    "Description": OPTIONAL,
+    "Options": {
+        "Values" : {
+            "v0" : "label0",
+            "v1" : "label1",
+            "v2" : "label2",
+            "v9" : "label9"
+        },
+        "Order" : {
+            "0" : "v9",
+            "1" : "v2",
+            "2" : "v1",
+            "3" : "v0"
+        },
+        "Orientation" : "horizontal|vertical",
+        "RequireResponse" : boolean
+    }
+}
+```
+
+`Type`: MUST be "radio".
+
+`Name`: Required. Follows `PageElement.Name` rules.
 
+`Description`: Optional. If not specified, the
+               radio buttons will be displayed with no accompanying text.
+               If specified, they will be displayed to the left of the buttons.
+               
+`Options.Values`: KEY REQUIRED, VALUE OPTIONAL. Contains a JSON object specifying options,
+                one of which may be selected. Each key/value corresponds
+                to `<radio value="JSONKey">JSONValue</radio>` in an
+                HTML implementation. The JSONKey contains the value
+                to be saved if selected, and the JSONValue contains
+                the human friendly text to display to the user.
+                NOTE: a JSONValue is OPTIONAL because, for a large set
+                of radio button questions, the values may be defined 
+                horizontally above the set. See `Options.Orientation`.
+                
+`Options.Order`: Optional. This specified the order values are assigned to
+                 rendered radio buttons. The `Options.Values` are going to be
+                 naturally ordered when encoded to JSON, so to preserve unnatural
+                 order, we must specify the real value order in this element.
+                 See example.
+
+`Options.Orientation`: REQUIRED. Rendering them vertically will display labels.
+                       Rendering them horizontally will require the use of a
+                       `radio-labels` element for context. See Section 2.1.7.
```

Comment:
Why make it required and not just specify a default?

Reply:
What should the default be?

Reply:
horizontal seems to be the more common use case for radio buttons on pages I can think of.